javascript_nodejs/81.facebook-events/README.MD

```diff
@@ -0,0 +1,53 @@
+[![Deploy to Azure](http://azuredeploy.net/deploybutton.png)](https://azuredeploy.net/)
```

Comment:
I don't think any other samples have this. Consider removing for now, and if relevant we'll add it in for all samples.